metadataclient/commands.py

```diff
@@ -985,9 +988,13 @@ def _normalize_human_friendly_time(val):
     _normalize_human_friendly_time.__doc__.format(_doc_ts_formats)
 )
 
-# TODO: Add when descriptor is None, use cached descriptor!
-# TODO: Fix descriptor replace (related to above)
-# TODO: Add server_disconnect that rolls all config back to default
-# TODO: Add capped collection caching layer for the client
-# TODO: Add fast read/write capped collection, different than caching capped collection
-# TODO: Add timeouts to servers in order to stop ppl from abusing data sources
+ins_dict = {'run_start': insert_run_start, 'run_stop': insert_run_stop,
+            'descriptor': insert_descriptor, 'event': insert_event,
+            'bulk_event': bulk_insert_events}
```

Comment:
That would add a dependency that MDC does not currently have, so strings are fine by me.